style/styles.css

```diff
@@ -341,3 +355,332 @@ textarea {
   background: url("../image/illustration-contact-form.png") 100% 77% no-repeat;
   background-attachment: scroll;
 }
+
+@media screen and (min-width: 900px) {
+  h1 {
+    margin: 15px 5px;
+  }
+
+  h2 {
+    width: auto;
+  }
+
+  /* header */
+  #welcome {
+    display: none;
+  }
+
+  #deskNav {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+  }
+
+  #navList {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    padding: 0;
+    align-content: center;
+    column-gap: 12px;
+  }
+
+  .navLink {
+    padding: 0;
+    font-size: 10px;
+    text-align: center;
+    text-decoration: none;
+    color: #172b4d;
+  }
+
+  #phoneNav {
+    display: none;
+  }
+
+  .about-me {
+    background: url("../image/Action.png") none;
+    background-size: 0;
+  }
+
+  /* section about me */
+  #main {
+    background: url("../image/Header-llustration-desktop.svg") no-repeat;
+    background-position: center;
+    background-size: cover;
+    height: 100vh;
+    padding-bottom: 100px;
+  }
+
+  #aboutBox {
+    width: 70%;
+    height: auto;
+  }
+
+  .me,
+  .whatIdo {
+    display: inline-block;
+    max-width: 95%;
+    max-height: auto;
+    font-weight: 800;
+    font-size: 40px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+  }
+
+  .paragrph {
+    max-width: 90%;
+    max-height: auto;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  #social-media {
+    margin-bottom: 50px;
+  }
+
+  /* recent  */
+  #indicator {
+    display: none;
+  }
+
+  .box-articles {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    column-gap: 15px;
+  }
+
+  #recent-works {
+    margin-bottom: 100px;
+  }
+
+  /* about me */
+  #about-me {
+    margin-bottom: 150px;
+    height: 100vh;
+  }
```

Comment:
Hello @Grandi0z,  I must commend that you have done wonderful work on your project and you made a great effort in getting the desktop version of the project done 👏 👏 🌹 . But I would kindly suggest we stick to the Figma template for the desktop version as much as possible. Currently, the about section of your project is having a `background image` that is not well placed and it is moving under the `Language` and `Framework` cards. Kindly see to fixing it before re-submitting for review

Figma template design 👇 

![abt real](https://user-images.githubusercontent.com/78634841/206579826-613fd53b-3bf7-408f-bf59-1ef339e70324.png)

Your Design 👇 

![poor](https://user-images.githubusercontent.com/78634841/206580173-0fa95eb8-186d-4372-a129-fb78a723a98c.png)